tests/baselines/reference/disallowThroughTypesConditionAndAvoidFallback(moduleresolution=nodenext).errors.txt

```diff
@@ -0,0 +1,28 @@
+error TS6504: File '/node_modules/foo/index.js' is a JavaScript file. Did you mean to enable the 'allowJs' option?

```

Comment:
i'm not sure how to avoid this in the test setup without `allowJs: true` (which I would prefer to avoid here).